### ISSUE TITLE
Chrome console error message

### ISSUE THREAD

**Issue by michelemazzali** [NUKnightLab/StoryMapJS#468]
Hi,
https://myportheritage.eu/exhibitions/ravenna-flux-of-people/
Chrome console fires an error
Uncaught TypeError: Cannot read property 'length' of undefined
firing at line 213 of StoryMap.js
```
 		// Emoji Support to Chrome?
		if (Browser.chrome) {
			loadCSS(urljoin(this.options.script_path,"../css/fonts/font.emoji.css"), function() {
			});
		}
```
thus impeding other js instructions to be executed (namely, a custom function which scrolls the div to another language).
Thanks!

**Comment by michelemazzali** [comment]
Update: I had to switch back to version 0.7.6 to make it work

**Comment by scott2b** [comment]
font.emoji.css is in the latest code in the CDN, so it is hard to know what is causing this error without seeing the code in it's broken state. The font file is loaded relative to what is determined to be the script path by the getScriptPath call in the buildStoryMap function. So, if you have a script tag on the page that interferes with the heuristics of that function (it uses an expression match to the script source reference) this could be one cause for this error, and one way I was able to replicate the error. Be sure your script tags are pointing to the CDN, or if you are hosting your own copy of the code, then you will also need to host relative assets like the emoji font file.

I have attached below a test.html which exemplifies creating a StoryMap page using the data from your StoryMap with the hope that you might be able to determine difference / potential conflicts with your own code that resulted in the error.

[test.html.gz](https://github.com/NUKnightLab/StoryMapJS/files/6678995/test.html.gz)
